src/SourceTokenList.ts

```diff
@@ -326,34 +379,37 @@ export default class SourceTokenList {
   /**
    * Allow iterating over the tokens in this list using e.g. `for (… of …)`.
    */
-  [Symbol.iterator]() {
+  [Symbol.iterator](): Iterator<SourceToken> {
     let index = this.startIndex;
-    let { endIndex } = this;
-    return (): { done: boolean; value: SourceToken | null | undefined } => {
```

Comment:
I think this was actually broken before since it didn't properly conform to the iterator protocol.